### DISCUSSION TITLE
How to we represent the Tractus-X CHANGELOG on the website

### DISCUSSION THREAD

**Discussion by SebastianBezold** [eclipse-tractusx/eclipse-tractusx.github.io#844] [General]
## Current Situation

The Tractus-X CHANGELOG is currently presented as a menu item in the "Versions" top-level navigation.
It is maintained as a Docusaurus page. The contents are manually copied from [eclipse-tractusx/tractus-x-release CHANGELOG](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md).
There is also a [configuration section](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/docusaurus.config.js#L152-L161), that targets the [eclipse-tractusx/tractus-x-release CHANGELOG](https://github.com/eclipse-tractusx/tractus-x-release/blob/main/CHANGELOG.md) as remote-content

## Potential Problems

Apparently, the CHANGELOG cannot be used as-is, since it will cause Docusaurus build errors. See comments and build history in #842. 
The behavior of the remote-content-plugin is not yet clear to me. When running a build locally, it actually deletes the file, instead of pulling the newest version. Maybe this is related to the build issue and it is doing a fail-safe?!
Another obvious problem is, that we could simply forget to update the CHANGELOG after a release, if we have to do it manually

## Suggestions

a) Remove the duplicated CHANGELOG from our website completely and instead create a page with general release information and a __Link__ to the CHANGELOG in tractus-x-release
b) Keep the duplicated CHANGELOG and remove the remote-content-plugin config. This would remove the kind of unknown behavior, but would keep the manual effort. Especially the manual adaptation to fix Docusaurus build errors

So I am in favor of a), but would put it to a vote

**Comment by evegufy** [comment]
option a 👍

**Reply by FaGru3n** [reply]
+1

**Reply by stephanbcbauer** [reply]
+1

**Reply by ClosedSourcerer** [reply]
+1
